notebook.ipynb

```diff
@@ -54,13 +54,13 @@
    "execution_count": null,
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        ("text-to-audio_audio_files", "audio_files1"),
</pre><p>leftovers of temporary file name <code>audio_files1</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demo-call-center/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>